docs/concepts/what-is-an-event.md

```diff
@@ -7,65 +7,144 @@ tags:
+Events are created by the Temporal Cluster in response to external occurrences and Commands generated by a Workflow Execution. Each Event corresponds to an `enum` that is defined in the [Server API](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/event_type.proto).
 
 All Events are recorded in the [Event History](/docs/concepts/what-is-an-event-history).
 
-The following is a complete list of all possible Events that could appear in a Workflow Execution Event History.
-
-Each Event corresponds to an `enum` that is defined in the [Server API](https://github.com/temporalio/api/blob/master/temporal/api/enums/v1/event_type.proto).
+A list of all possible Events that could appear in a Workflow Execution Event History is provided below.
 
 ### WorkflowExecutionStarted
 
-This is always the first Event in a Workflow Execution History and it indicates that the execution was invoked.
+This is always the first Event in a Workflow Execution History. It indicates that the execution was invoked.
+
+| Field | Type | Definition |
+| ----- | ---- | ---------- |
 
 ### WorkflowExecutionCompleted
 
-This event type indicates that the Workflow Execution has successfully completed and contains Workflow Execution results.
+This indicates that the Workflow Execution has successfully completed. The Event contains Workflow Execution results.
+
+| Field                            | Type     | Definition                                                                        |
+| -------------------------------- | -------- | --------------------------------------------------------------------------------- |
+| result                           | Payloads | Serialized result of completed Workflow.                                          |
+| workflow_task_completed_event_id | int64    | The Id of the 'WORKFLOW_TASK_COMPLETED' that the event command was reported with. |
+| new_execution_run_id             | string   | The Id of the new workflow started by cron.                                       |
 
 ### WorkflowExecutionFailed
 
-This event type indicates that the Workflow execution has unsuccessfully completed and contains the Workflow execution error.
+This event indicates that the Workflow execution has unsuccessfully completed and contains the Workflow execution error.
+
+| Field                            | Type       | Definition                                                                        |
+| -------------------------------- | ---------- | --------------------------------------------------------------------------------- |
+| failure                          | Failure    | Serialized result of a Workflow failure, such as an exception or error.           |
+| retry_state                      | RetryState |                                                                                   |
+| workflow_task_completed_event_id | int64      | The Id of the 'WORKFLOW_TASK_COMPLETED' that the event command was reported with. |
```

Comment:
```suggestion
| workflow_task_completed_event_id | int64      | The Id of the `WORKFLOW_TASK_COMPLETED` that the event command was reported with. |
```

Add back ticks when referencing code not apostrophes, so the code is formatted.

Reply:
In this case though, WORKFLOW_TASK_COMPLETED is actually the Event WorkflowTaskCompleted - and we want to link to them I think